lib/matplotlib/axes/_axes.py

```diff
@@ -3153,7 +3153,7 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
            If True, will toggle on the rendering of the caps
 
         showbox : bool, default = True
-           If True, will toggle on the rendering of the box
+           If True, will toggle on the rendering of box
```

Comment:
Woops, looks like I accidentally used an older _axes.py file when making the changes.